tables/th-g0-backward.utb

```diff
@@ -0,0 +1,222 @@
+# liblouis: Thai Current Braille System.
+#
+#-index-name: Thai
+#-display-name: Thai current braille
+#
+#+language:th
+#+region:th-TH
+#+type:literary
```

Comment:
Should this table be labeled "computer braille" instead of literary?

Reply:
yes

Reply:
I changed it